commit 97c5690abeaa88767acf2ffbb55552e8278052c8
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Mon Jan 11 03:34:50 2010 +0100

    Fix tmpfs assertion
    
    	* tmpfs/tmpfs.h (tmpfs_dirent): Add padding field to push the
    	name field after its position in struct dirent.

diff --git a/tmpfs/tmpfs.h b/tmpfs/tmpfs.h
index 3032ce3..a0e1f7a 100644
--- a/tmpfs/tmpfs.h
+++ b/tmpfs/tmpfs.h
@@ -64,6 +64,7 @@ struct tmpfs_dirent
 {
   struct tmpfs_dirent *next;
   struct disknode *dn;
+  uint32_t pad;
   uint8_t namelen;
   char name[0];
 };
